DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetAlternativeCareersAndSkillsForOccupation.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "query": "MATCH (o:esco__Occupation {uri:{occupation}})-->(s:esco__Skill)<--(o2:esco__Occupation) MATCH (o2)-[es:esco__relatedEssentialSkill]->(s) WITH o, o2, COUNT(es) as MatchingEssentialSkillsCount MATCH (o2)-[os:esco__relatedOptionalSkill]->(s) WITH o, o2, MatchingEssentialSkillsCount, COUNT(os) as MatchingOptionalSkillsCount MATCH (o2)-[:esco__relatedEssentialSkill]->(s) WITH o2, MatchingEssentialSkillsCount, MatchingOptionalSkillsCount, COUNT(*) as TotalEssentialSkillsCount MATCH (o2)-[:esco__relatedOptionalSkill]->(s) WITH o2, MatchingEssentialSkillsCount, MatchingOptionalSkillsCount, TotalEssentialSkillsCount, COUNT(*) as TotalOptionalSkillsCount WITH { uri:o2.uri, occupation:head(o2.skos__prefLabel), MatchingEssentialSkills:MatchingEssentialSkillsCount, MatchingOptionalSkills:MatchingOptionalSkillsCount, TotalEssentialSkills:TotalEssentialSkillsCount, TotalOptionalSkills:TotalOptionalSkillsCount, lastModified:head(o2.dct__modified) } as occupationsList MATCH (o:esco__Occupation {uri:{occupation}})-->(s:esco__Skill) MATCH (o)-[es:esco__relatedEssentialSkill]->(s) WITH occupationsList, o, COUNT(es) as totalEssentialSkillsCount MATCH (o)-[os:esco__relatedOptionalSkill]->(s) WITH occupationsList, o, totalEssentialSkillsCount, COUNT(os) as totalOptionalSkillsCount WITH { uri:o.uri, occupation:head(o.skos__prefLabel), alternativeLabels:o.skos__altLabel, lastModified:o.dct__modified, totalEssentialSkills:totalEssentialSkillsCount, totalOptionalSkills:totalOptionalSkillsCount, occupations:collect(occupationsList)} as occupation RETURN occupation",
```

Comment:
Spec calls for a percentage match, whereas this query returns a count.

Reply:
The matching skills count appears incorrect anyway.

E.g. occupation with uri `http://data.europa.eu/esco/occupation/0f845e64-de8b-431e-9fe3-1bccf8e1c5b9` has 8 optional skills, but when you run this query, the first alternative occupation (`"http://data.europa.eu/esco/occupation/27eb624c-6171-4c71-aa71-bb21e83381d4"`) is reporting `matchingOptionalSkills` as 15.

15 is actually the number of optional skills for that occupation. Looking at the optional skills for both occupations, there are no common option skills, so we should be returning a percentage of 0. (Or as-is where it's returning a count, it should also be 0.)